resources/exercise-markdown.Rmd

```diff
@@ -0,0 +1,50 @@
+---
+title: "Using Markdown"
+output:
+  html_document:
+    self_contained: true
+---
+
+Markdown is **great**! 
+
+## Why bother?
+
+Markdown *reduces* the time and effort spent between writing analysis and writing
+the text for the document, since it's all in the same place. Plus, it's a good
+first step to being more reproducible in your work!
+
+### Being reproducible
+
+Reproducibility is a cornerstone of science. A scientific result is reproducible
+when:
+
+1. You take the **same** data
+2. You take the **same** code/analysis steps
+3. Get someone else to run your analysis
+4. The results they get are the **same** as the results presented in your paper
+
+Sound easy? It's actually surprising more difficult that it seems.
+
+## Why it can be fun!
+
+With the single formatting of Markdown you can create:
+
+- Multiple types of slides
+  - Like the really nice [xaringan](https://slides.yihui.name/xaringan/#1)
+- [Dashboards](https://rmarkdown.rstudio.com/flexdashboard/)
+- Books, blogs
+- Manuscripts
+  - And can change the template based on the journal with [rticles](https://github.com/rstudio/rticles)!
+
+And many more output documents! 
+
+## Compare to Word
+
+Let's compare to Word:
+
+| Task | Word | R Markdown |
+|:--|:--|:--|
+| Inserting results | Hard | Easy |
+| Collaborating with others | Others used to it, so could be easy | New and modern, likely unfamiliar, so can be difficult |
+| Change reference style and bibliography | Tricky and buggy | Easy |
```

Comment:
Nope - changing reference style in Word is as easy as adding a different template. (I speak as someone who spent several hours trying to fix bibliography formatting to conform to Taylor & Francis's spec with .csl files.)

Reply:
I personally don't use word at all for these types of things, but several of my friends complain about "referencing" in Word being difficult, so I was only using their experiences as a basis for this statement. I'll remove and simplify this section a bit.